api/create.go

```
@@ -298,3 +299,25 @@ func getAppName(appName, appJson string) (string, error) {
 
 	return appName, nil
 }
+func GetTempDir() string {
+	err := os.Setenv("FLOGO_BUILD_EXPERIMENTAL", "true")
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = os.Setenv("GO111MODULE", "on")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer os.Unsetenv("FLOGO_BUILD_EXPERIMENTAL")
+
+	tempDir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		log.Fatal(err)
```

This method should probably return err instead of eating it and logging the error